web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
In fact, in the name of the pragmatism, we can continue as this case is very trivial, but extending this practice across all your contributions will lead to bloated files like the Odoo ones, and new contributors that examine other modules to copy practices will be very confused with the heterogeneity.

Anyway, the final word is in the reviewers or the PSC, so in l10n-brazil you can agree another criteria skipping one of the rules/guidelines, but please in general repos like this one try to keep the rule.